### ISSUE TITLE
Data Viewer window error

### ISSUE THREAD

**Issue by glenmwright** [erykjj/jwlmanager#112]
Good day, 

In version 4.0 getting the Error, "Oops! Something went wrong". The error appears when you close the Data Viewer window but not all the time.  

```
JWLManager v4.0.0
Windows-10-10.0.19045-SP0

Traceback (most recent call last):
  File "JWLManager.py", line 1871, in data_viewer
RuntimeError: Internal C++ object (DataViewer) already deleted.
```

**Comment by erykjj** [comment]
Hopefully resolved by 5ffb643170491b37d7e82365f0db7a7aab6afe70 in [v4.0.1](https://github.com/erykjj/jwlmanager/releases/tag/v4.0.1)
Please confirm.

**Comment by glenmwright** [comment]
Yes, it appears to be resolved. For reference, I believe the crash occurred for notes not linked to a publication. 

![image](https://github.com/erykjj/jwlmanager/assets/85457329/993b38b0-d123-4478-af3b-fb29410fdacc)
